addons/mrp/models/product.py

```diff
@@ -30,6 +30,14 @@ def _compute_used_in_bom_count(self):
             template.used_in_bom_count = self.env['mrp.bom'].search_count(
                 [('bom_line_ids.product_id', 'in', template.product_variant_ids.ids)])
 
+    def write(self, values):
+        super().write(values)
```

Comment:
The super write function do not return None. So it's a good practice to catch it's return and return it after.
res = super().write(values)
return res